Dockerfile

```diff
@@ -1,12 +1,8 @@
 FROM jujusolutions/jujubox:latest
+MAINTAINER Charles Butler <charles.butler@canonical.com>
 
-VOLUME "/home/ubuntu/.juju"
-VOLUME "/home/ubuntu/trusty"
-VOLUME "/home/ubuntu/builds"
-VOLUME "/home/ubuntu/layers"
-VOLUME "/home/ubuntu/interfaces"
-ADD install-review-tools.sh /install-review-tools.sh
-RUN /install-review-tools.sh
-# Override jujubox run.sh
-ADD run.sh /run.sh
-CMD /run.sh
+ENV LAYER_PATH=/home/ubuntu/charms/layers
```

Comment:
This is a default. I guess I was assuming the user would override this export. Thing is, there's no clear indicator to them that this needs to happen, so I concede the point that this is a bad default value. I'll take a look at args and see if we can scrape that $USER arg out of there and re-use in the ENV stanza's